Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserWidget.cpp

```diff
@@ -653,7 +653,7 @@ void ezQtAssetBrowserWidget::OnTransform()
     ezUuid guid = m_pModel->data(index, ezQtAssetBrowserModel::UserRoles::AssetGuid).value<ezUuid>();
     QString sPath = m_pModel->data(index, ezQtAssetBrowserModel::UserRoles::RelativePath).toString();
     range.BeginNextStep(sPath.toUtf8());
-    auto res = ezAssetCurator::GetSingleton()->TransformAsset(guid, false);
+    auto res = ezAssetCurator::GetSingleton()->TransformAsset(guid, ezTransformFlags::None);
```

Comment:
From where is this called ? If this is right-click -> transform in the asset browser, I would probably also add ManuallTransformed here.

Reply:
Done.